src/components/FormSelectionContainer/FormSelection/FormSelection.tsx

```diff
@@ -0,0 +1,56 @@
+import styled from "@emotion/styled";
+import React from "react";
+import { Config, Form } from "../../../types";
+import { Button } from "../../../UI/Button";
+import { Title } from "../../../UI/Title";
+import { Wrapper } from "../../../UI/Wrapper";
+import { Container } from "../../Container";
+import { ProgressBar } from "../../ProgressBar";
+
+interface FormSelection {
+  config: Config;
+}
+
+export const FormSelection = styled(({ className, config }) => {
+  // eslint-disable-next-line @typescript-eslint/explicit-function-return-type
+  const selectedForm = (form: string) => {
+    console.log(form);
+  };
+
+  return (
+    <Container>
+      <div className={className}>
+        <Wrapper>
+          <ProgressBar step={2} />
+          <Title>Choose Document Type to Issue</Title>
+          <div className="buttonWrapper">
+            {config.forms.map((form: Form, i: number) => {
+              return (
+                <Button
+                  data-testid={`${form.type}-button`}
+                  key={i}
+                  onClick={() => selectedForm(form.name)}
+                >
+                  {form.name}
+                </Button>
+              );
+            })}
+          </div>
```

Comment:
```suggestion
          <div className="flex flex-wrap -mx-2">
            {config.forms.map((form: Form, i: number) => {
              return (
                <div key={i} className="w-full mb-2 lg:w-1/3 px-2 lg:mb-0">
                  <Button
                    data-testid={`${form.type}-button`}
                    onClick={() => selectedForm(form.name)}
                  >
                    {form.name}
                  </Button>
                </div>
              );
            })}
          </div>
```
since we are using tailwind already, if possible try to avoid custom styles (buttonWrapper), also can try use the way responsive column works in tailwind way https://tailwindcss.com/components/flexbox-grids/#column-spacing

Reply:
oohhh ok will change the custom style to use tailwind